vmm/src/lib.rs

```diff
@@ -1221,18 +1111,15 @@ impl Vmm {
     }
 
     fn start_microvm(&mut self) -> std::result::Result<VmmData, VmmActionError> {
-        START_INSTANCE_REQUEST_CPU_TS.store(now_cputime_us() as usize, Ordering::Release);
```

Comment:
Why can't we keep `START_INSTANCE_REQUEST_CPU_TS` and `START_INSTANCE_REQUEST_TS` as they were before? They're still accessible from `vstate`, we wouldn't have to pass them around, and they can also be used from other places should that ever matter.

Reply:
I was aiming for getting rid of global state since it's not in any way necessary for these to be global.

There should never be *other places* that they'd be needed in.

I think it's more in lines with the general model of Firecracker's codebase to *move* them to where they're needed rather than keeping them as global state. Logical flow is that the `vmm` thread initializes the TS and then it's *moved/consumed* by the `vcpu`s who are then the sole users of the TS local copies.

Reply:
Insisting on not using global vars especially when it can be so easily avoided.